src/cpptools.ts

```diff
@@ -65,7 +65,9 @@ interface TargetDefaults {
 
 function parseCppStandard(std: string, can_use_gnu: boolean): StandardVersion {
     const is_gnu = can_use_gnu && std.startsWith('gnu');
-    if (std.endsWith('++2a') || std.endsWith('++2b') || std.endsWith('++20') || std.endsWith('++latest')) {
+    if (std.endsWith('++23') || std.endsWith('++2b') || std.endsWith('++latest')) {
```

Comment:
After updating `package.json` to get version `6.1.0` of the API, we should only allow c++23 if the API verison is v6 or higher.  There is a boolean parameter to this function `can_use_gnu` that is based on a similar version check.  Perhaps we should just pass the version into this function and create the local `can_use_gnu` variable here instead of in `parseCompileFlags`.

Reply:
I added the check for v6 or higher.

I think it's better to keep `can_use_gnu` inside of `parseCompileFlags` because it is also passed into `parseCStandard` and we wouldn't want to duplicate the logic in both functions.

Feel free to push changes directly to my branch if that's easier for you. Or I can make changes based on your suggestions - either way is fine with me.

Reply:
This is fine.  I hadn't looked closely at the other function before making the suggestion.

Reply:
Oh, it looks like `package.json` was not updated yet.  I'll make that change real quick.

Reply:
Thanks for all your help!

Reply:
Thanks for taking the initiative to start this.